app/src/test/java/android/util/Log.java

```diff
@@ -0,0 +1,23 @@
+package android.util;
```

Comment:
Why is this file in our project ?

Reply:
So we can use unitary tests for code using the Android Log infrastructure instead of being coerced to do instrumented tests (which take much much longer than junit tests).